staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -576,6 +576,8 @@ type KubeletConfiguration struct {
 	// evictionMaxPodGracePeriod is the maximum allowed grace period (in seconds) to use
 	// when terminating pods in response to a soft eviction threshold being met. This value
 	// effectively caps the Pod's terminationGracePeriodSeconds value during soft evictions.
+	// Negative values are not supported and do not defer to the Pod's value;
```

Comment:
instead of saying `and do not defer to the Pod's value` please write what it is deferred to and what the behavior will be